engines/web/app/controllers/web/application_controller.rb

```diff
@@ -0,0 +1,5 @@
+module Web
+  class ApplicationController < ActionController::Base
```

Comment:
So is the purpose of this to allow access from within the engine to the helpers attached to the main ApplicationController?

Reply:
yeah, this is a separate app, I dont think we will inherit from the main app app_controller. We want this app to be a different and separate app. We will need to see what we do what common code, but I dont think it should be through extending the main app_controller.